api-docs/cloud-load-balancers-v1/release-notes/releases/clbs-v1-26-23-20190911.rst

```diff
@@ -0,0 +1,18 @@
+.. version-v1.26.17-release-notes:
+
+v1.26.23, Sep 18th, 2019
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+What's new
+-----------
+- Updated Atlas Backend Adapter to a more fault tolerant interface.
```

Comment:
Remove period at end of line 8 (to be consistent with what you have in Resolved issues and in other release notes).